### ISSUE TITLE
Jobs List - Create skeleton

### ISSUE THREAD

**Issue by mabashian** [ansible/awx#4226]
## Jobs List

Mockup: https://tower-mockups.testing.ansible.com/patternfly/jobs/jobs/

This view should duplicate existing AWX behavior and display a list of all (non-sync) jobs. This list should be populated from `/api/v2/unified_jobs/` and filter by `order_by=-finished&not__launch_type=sync`.

#### Initial Skeleton Tasks
* Hookup Route
* Configure Breadcrumb 
* Handle Empty State
* Pagination
* Sort hookup is out of scope for this issue and will be handled by https://github.com/ansible/awx-pf/issues/226
* Delete hookup is out of scope for this issue and will be handled by https://github.com/ansible/awx-pf/issues/227
* Each row should display the job ID, name, status and type
* Sparkline used to show job status should be completed at this point, if not then this will have to wait #224 
* Job ID/name should link to job results view #219 (this may need to be peeled out into it's own issue if the job results page DNE yet) 
* Mark strings for translation
* Responsiveness

#### Not in Scope for MVP
* Resource missing icon (I think this applies specifically to workflows where templates have been deleted)
* Relaunch icon and functionality
* Anything related to workflow rows (we want to display the workflow job id/name/status/type but nothing else).  The mockups indicate an expand/collapse to see all the workflow node jobs.  That is not in scope for the MVP.

<img width="1476" alt="Screen Shot 2019-05-23 at 4 31 05 PM" src="https://user-images.githubusercontent.com/9889020/58284242-30d00f00-7d78-11e9-8429-845adadfa673.png">

**Comment by mabashian** [comment]
Note that this blocks work on #4228 and #4227.  Once this lands we can remove the blocked label from those issues.

**Comment by jakemcdermott** [comment]
closing - resolved by: https://github.com/ansible/awx/commit/d8452e1259347cd9efeb3ae34e936bc3106f24b5, https://github.com/ansible/awx/commit/69426dee0807526e078e88dfa2c70d44a6f68ee5
